.github/actions/bpftrace/scripts/check-ipsec-leaks.bt

```diff
@@ -0,0 +1,223 @@
+// Six parameters expected:
```

Comment:
Definitely nit, but if you want syntax highlighting here, see https://github.com/bpftrace/bpftrace/blob/master/.gitattributes.